r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1732,3 +1732,44 @@ test_that("parse_date_time() doesn't work with hour, minutes, and second compone
     '"ymd_HMS" `orders` not supported in Arrow'
   )
 })
+
+test_that("year, month, day date/time parsers work", {
+  test_df <- tibble::tibble(
+    ymd_string = c("2022-05-11", "2022/05/12", "22.05-13"),
+    ydm_string = c("2022-11-05", "2022/12/05", "22.13-05"),
+    mdy_string = c("05-11-2022", "05/12/2022", "05.13-22"),
+    myd_string = c("05-2022-11", "05/2022/12", "05.22-14"),
+    dmy_string = c("11-05-2022", "12/05/2022", "13.05-22"),
+    dym_string = c("11-2022-05", "12/2022/05", "13.22-05")
+  )
+
+  # string processing requires RE2 library (not available on Windows with R 3.6)
```

Comment:
My immediate reaction: but this isn't string processing, this is datetime processing!

```suggestion
  # these functions' internals use some string processing which requires RE2 library (not available on Windows with R 3.6)
```

Reply:
We're parsing a string into a date or a date/time. Some string processing takes place before a call to `strptime()`.

Reply:
Feel free to rephrase my suggestion, but the phrasing still could be clearer.

Reply:
Having said that ☝🏻 I like your suggestion as it makes things a bit clearer, so I've added it.